docs/source/tutorials/07_using_model_hub_nodes.rst

```diff
@@ -0,0 +1,334 @@
+.. include:: /include/substitution.rst
+
+*********************
+Using Model Hub Nodes
+*********************
+
+PeekingDuck has support for external model hub models. You can leverage the model
+hub nodes to use these external models with the PeekingDuck pipeline. The model
+hub nodes differ in design from traditional model nodes as the model hub models
+perform a variety of computer vision tasks. This tutorial demonstrates how
+to use these nodes through some sample pipelines.
+
+
+List of Model Hub Nodes
+=======================
+
+The table below shows the model hub nodes available.
+
++-----------------------------------------------------------------+---------------------------------+
+| Model                                                           | Documentation                   |
++=================================================================+=================================+
+| `Hugging Face Hub <https://huggingface.co/docs/hub/index>`_     | :mod:`model.huggingface_hub`    |
++-----------------------------------------------------------------+---------------------------------+
+| `MediaPipe Solutions <https://google.github.io/mediapipe/>`_    | :mod:`model.mediapipe_hub`      |
++-----------------------------------------------------------------+---------------------------------+
+
+
+Hugging Face Hub
+================
+
+The :mod:`model.huggingface_hub` node supports transformer models which perform
+the following computer vision tasks:
+
+   #. Instance segmentation
+   #. Object detection
```

Comment:
Suggest swapping the order to `Object detection` first, then `Instance Segmentation` (and reorder relevant sections below similarly).
'coz object detection is the "basic" task that PKD first supports, then came instance segmentation.
Also, the latter is a refined version of the former:  in the screenshots, the former blurs the bbox (rough outline) while latter blurs the object (more focused).

Reply:
Swapped the order and rephrased the intoduction as well